src/hotspot/cpu/riscv/nativeInst_riscv.hpp

```diff
@@ -111,41 +111,52 @@ NativeCall* nativeCall_before(address return_address);
 // The NativeCall is an abstraction for accessing/manipulating native
 // call instructions (used to manipulate inline caches, primitive &
 // DSO calls, etc.).
+// On riscv, NativeCall is a reloc call.
```

Comment:
Suggestion: `NativeCall is reloc call on RISC-V. See MacroAssembler::reloc_call`